### PR TITLE
[Tensorflow][Build]Adding Major label to TF1 Images

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*
Adding major labels to all Tf1.15 Images

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/inference/docker/1.15.3/py3/Dockerfile.gpu [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM nvidia/cuda:10.0-base-ubuntu18.04
 
 LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="4"
```
Comment (YYStreet) [inline tensorflow/inference/docker/1.15.3/py3/Dockerfile.gpu:4]: We do not have inference py37 images for tf1.15

File: tensorflow/training/docker/1.15.3/py3/Dockerfile.gpu [hunk 2]
```diff
@@ -3,6 +3,7 @@
 FROM nvidia/cuda:10.0-base-ubuntu18.04
 
 LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="4"
```
Comment (YYStreet) [inline tensorflow/training/docker/1.15.3/py3/Dockerfile.gpu:6]: The training py37 TF 1.15.2 image in prod repo is with version 7.0

File: tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu [hunk 3]
```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:18.04
 
 LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="4"
```
Comment (YYStreet) [inline tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu:4]: The training py37 TF 1.15.2 image in prod repo is with version 7.0

File: tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu [hunk 4]
```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:18.04
 
 LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="4"
```
Comment (YYStreet) [inline tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu:4]: We do not have inference py37 images for tf1.15

File: tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu (hunk 3 above)
Comment (TusharKanekiDey) [reply tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu:4]: Valid point. Will make the change.

File: test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py [hunk 5]
```diff
@@ -18,7 +18,7 @@
 TF_ADDONS_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testTFAddons")
 
 TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p2.8xlarge", processor="gpu")
-TF_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.4xlarge", processor="cpu")
+TF_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="m4.16xlarge", processor="cpu")
```
Comment (TusharKanekiDey) [inline test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py:21]: By mistake, removing this

Comment (saimidu) [inline test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py]: Why this change?

**Comment by TusharKanekiDey** [comment]
> Are you going to remove py37 inf images in a separate PR?

Yes, I will remove that as a part of different PR.
